test_scripts/API/VehicleData/StabilityControlsStatus/030_OnVehicleData_StabilityControlsStatus_ignition_ON_OFF.lua

```diff
@@ -0,0 +1,44 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0253-New-vehicle-data-StabilityControlsStatus.md
+--
+-- Description: Check receiving StabilityControlsStatus data via OnVehicleData notification
```

Comment:
@LitvinenkoIra I would propose to specify that it is checked of receiving StabilityControlsStatus data via OnVehicleData notification after mobile application data resumption.